documentation/library_guide/api_for_sycl_kernels/tested_standard_cpp_api.rst

```diff
@@ -456,8 +456,7 @@ These tests were done for the following versions of the standard C++ library:
 libstdc++ (GNU)                               Provided with GCC* 8.4.0, GCC 9.3.0,
                                               GCC 11.4.0, GCC 13.2.0
 --------------------------------------------- ---------------------------------------------
-libc++ (LLVM)                                 Provided with Clang* 11.0, Clang 14.0,
-                                              Clang 17.0, Clang 18.1
+libc++ (LLVM)                                 Provided with Clang* 20.1
 --------------------------------------------- ---------------------------------------------
 Microsoft Visual C++* (MSVC) Standard Library Provided with Microsoft Visual Studio 2019
                                               and Microsoft Visual Studio 2022.
```

Comment:
I think we need to extend the table to also indicate which SYCL compiler(s) we used for testing.

Reply:
Extended.